neurodsp/sim/aperiodic.py

```diff
@@ -119,64 +119,61 @@ def sim_synaptic_current(n_seconds, fs, n_neurons=1000, firing_rate=2.,
 
 @normalize
 def sim_knee(n_seconds, fs, chi1, chi2, knee):
-    """Returns a time series whose power spectrum follows the Lorentzian equation:
-
-    P(f) = 1 / (f**chi1 * (f**chi2 + knee))
-
-    using a sum of sinusoids.
+    """Simulate a signal whose power spectrum has a 1/f structure with a knee.
 
     Parameters
-    -----------
+    ----------
     n_seconds: float
-        Number of seconds elapsed in the time series.
+        Simulation time, in seconds.
     fs: float
-        Sampling rate.
+        Sampling rate of simulated signal, in Hz.
     chi1: float
         Power law exponent before the knee.
     chi2: float
         Power law exponent added to chi1 after the knee.
     knee: float
-        Location of the knee in hz.
+        Location of the knee in Hz.
 
     Returns
     -------
-    sig: 1d array
-        Time series with desired power spectrum.
+    sig : 1d array
```

Comment:
If white space is preferred after the return/arg, it could also be added to the parameters above.